Modelica/Fluid/Examples/AST_BatchPlant.mo

```diff
@@ -2252,7 +2225,7 @@ Implemented trace substances and missing equation for outflow of multi substance
   annotation (preferredView="info",Documentation(info="<html>
 <p>
 The process under consideration is an evaporation plant for a
-student lab at the Process Control Laboratory (AST) of the
+student lab at the Process Control Laboratory (AST for \"Anlagensteuerungstechnik\" in German) of the
```

Comment:
Even if a bit overlapping, should the English translation of 'AST' as 'Plant control technology' as well? It's clearly related to "Process Control Laboratory"; but still different.

Reply:
Right. The Process Control Laboratory (AST or Chair of Anlagensteuerungstechnik) was renamed to Process Dynamics and Operations Group (DYN). You can read about their 30 years of history [here](http://www.dyn.bci.tu-dortmund.de/cms/en/group/news/2020/30th-Anniversary-of-the-DYN-Group/index.html) and find the student lab as DYN 6a [here](http://www.dyn.bci.tu-dortmund.de/cms/en/teaching/laboratories/index.html).